src/applications/find-forms/components/SearchResult.jsx

```diff
@@ -70,6 +70,40 @@ const recordGAEventHelper = ({
     'search-typeahead-enabled': false, // consistently populate with false since there's no type ahead enabled for this search feature
   });
 
+const deriveRelatedTo = ({
```

Comment:
I'm digging the word `derive` here haha I feel like @ncksllvn and I also use the same language often! 🔥